assets/chezmoi.io/docs/reference/templates/delimiters.md

```diff
@@ -0,0 +1,23 @@
+# Delimiters
+
+By default, chezmoi uses the standard `text/template` delimiters `{{` and `}}`.
+If a template contains the string
+
+    chezmoi:template:left-delimiter=$LEFT right-delimiter=$RIGHT
+
+Then the delimiters `$LEFT` and `$RIGHT` are used instead. chezmoi will remove
+the `$LEFT` and `$RIGHT` strings from the line in the template before template
+execution to avoid parse errors. Either or both of `left-delimiter=$LEFT` and
+`right-delimiter=$RIGHT` may be omitted. `$LEFT` and `$RIGHT` must be quoted if
+they contain spaces. If either `$LEFT` or `$RIGHT` is empty then the default
+delimiter (`{{` and `}}` respectively) is set instead.
```

Comment:
Two things worth noting:

1. Only the *first* instance of the delimiter directive means anything.
2. Because of the way that chezmoi loads and parses templates, each template's delimiters must be specified individually. They are *not* inherited.

Reply:
Thanks, done.